src/js/mixins/DuplicableRowsMixin.jsx

```diff
@@ -40,7 +40,7 @@ var DuplicableRowsMixin = {
     Object.keys(duplicableRowsScheme).forEach(function (fieldId) {
       if (state.rows[fieldId] == null || state.rows[fieldId].length === 0) {
         let rowScheme = Util.extendObject(duplicableRowsScheme[fieldId], {
-          consecutiveKey: Util.getUniqueId()
+          consecutiveKey: parseInt((Util.getUniqueId() + "").slice(-9))
```

Comment:
Well the unique ID is actually just a timestamp of milliseconds since 1970. Which means that we have a timeframe of the last nine digits of the timestamp. This is 999,999,999 / (1000 * 60)  minutes until the numbers repeat. This is 11.57 days. The only real thread for duplicated unique id's are if the Application is running 11.57 days and in a exact millisecond another field is created. I think that is a chance I am happy to take since the consecutive key is only used while one cycle of modal is open. And it is only for one "type"